app/template/rest.py

```diff
@@ -98,6 +98,7 @@ def create_template(service_id):
         errors = {'content': [message]}
         raise InvalidRequest(errors, status_code=400)
 
+    # TODO clean up validator, use class method instead.
```

Comment:
Please create a ticket for this TODO and add the ticket number to this comment.

Reply:
https://github.com/department-of-veterans-affairs/notification-api/issues/1410

Reply:
Can you add the ticket number to the comment in the file, please?
`# TODO 1410 clean up validator, use class method instead.`

Reply:
good idea. done